### DISCUSSION TITLE
How do I contribute with language translation?

### DISCUSSION THREAD

**Discussion by ziggekatten** [apache/superset#25163] [General]
Simple question, what is the steps needed for me to create a swedish translation of Superset?

This would be my first participation as an contributor in an OSS project.

**Comment by ovelindstrom** [comment]
I would like to assist on the Swedish translation. Have you made a fork yet @ziggekatten?

**Comment by ovelindstrom** [comment]
Steps to create a new translation is found at https://superset.apache.org/docs/contributing/translations/.
